docs/reference/api/platform/Assert.md

```diff
@@ -1,11 +1,33 @@
 ## Assert
 
-[Add description here.]
+Mbed OS provides a set of macros that evaluates an expression and prints an error message if the expression evaluates to false. There are two types of macros, one for evaluating the expression during runtime and one for compile-time evaluation. `mbed_assert.h` defines these macros.
 
-### Assert class reference
+### Assert macros reference
 
-[Embed class reference here. If you've never done this, please ask your editor.]
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/group__platform__Assert.html)
 
-### Assert example
+### Assert usage example
 
-[Add example(s) here.]
+You can use the `MBED_ASSERT` macro for runtime evaluation of expressions. If the evaluation fails, an error message prints to bash in the below format. Note that the `MBED_ASSERT` macro is available in the debug and develop build profiles but not in the release build profile. 
+
+mbed assertation failed: <EVALUATED EXPRESSION>, file: <FILE NAME>, line <LINE NUMBER IN FILE>
```

Comment:
Query: Should this be part of the code block that starts on line 17? If not, please put it in code format and give it some context.

Reply:
Hi Amanda, I fixed the comments but kept the heading as "Assert" since we are explaining the usage/purpose in the doc anyway, also I thought if we change to Mbed Assert, the same applies to many other places as well, so I thought we will keep the heading as Assert. Let me know if you think otherwise.